arch/x86_64/mcount-dynamic.c

```diff
@@ -441,10 +441,10 @@ bool add_bad_jump(struct mcount_dynamic_info *mdi, unsigned long callsite,
 		return true;
 
 	sym = find_sym(&mdi->map->mod->symtab, target - mdi->map->start);
+	/* only care about jumps to the middle of a function */
 	if (sym == NULL)
-		return true;
+		return false;
```

Comment:
This is just to be conservative since we don't know it's correct or not.  Let's make it robust even if it cannot trace some.

Reply:
@namhyung  I think this is reasonable. because we can not patch the function without symbol. so, jumping to not recognized address in symbol no need to exclude.